std_msgs/Header header
uint8 status
uint8 fault_code
uint8 stbs_status